docs/components/movement-sensor/wheeledodometry.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Configure a wheeledodometry movement sensor"
+linkTitle: "wheeledodometry"
+type: "docs"
+description: "Configure a wheeledodometry movement sensor."
+images: ["/icons/components/imu.svg"]
+tags: ["movement sensor", "components", "movement sensor"]
+# SMEs: Rand, Martha
+---
+
+Configure a `wheeledodometry` movement sensor to implement _wheeled odometry_ on your robot.
+
+_Wheeled odometry_ is the estimation of position, orientation, linear velocity, and angular velocity using the dimensions of a base.
+This model uses [encoders](/components/encoder/) to get an odometry estimate from an encoder wheeled base.
+
+## Set-up requirements
+
+To prepare your robot, attach [encoders](/components/encoder/) to each the motors of your base to measure their rotation.
+
+- Configure your rover as a [wheeled base component](/components/base/wheeled/).
+- Configure each of these encoded motors [as encoder components](/components/encoder/#configuration).
```

Comment:
these are motor components. They have to be position reporting.

Reply:
Oh, ok! Will update to reflect and comment here when done. What kinds of motors specifically will work? Is it just https://docs.viam.com/components/motor/gpio/ or am I confused?

Reply:
Think I updated but happy to change all around again if it's not just GPIO or isn't GPIO-- apologies for the confusion!